### PR TITLE
Logging bundle

### PR DESCRIPTION
**Pull request by joker806**
Zatim obsahuje Monolog processor, ktery provaze logy s tracemama v DD. Do budoucna bych rad rozsiril o veci, ktere ted mame v `api-error-control` (s tim jak budeme prechazet s logama a error trackingem na DD).

Diky tomu, ze je to bundle, staci to nainstalovat do projektu a neni potreba delat nic dalsiho (services si to zaregistruej samo, nevyzaduje zadnou konfiguraci).

Ozkousel jsem si tu nekolik veci najednou:
* jak slozite je ted vykopnout novou lib - max hodina (kdyz by se udelal nejaky skeleton, muze byt i vyrazne min)
* jak slozite je udelat Symfony bundle, ktery nebude potreba slozite konfigurovat - relativne easy, akorat se to blbe testuje (par let uz jsme bundle nezakladal, tak to byl trosku research)

### CODE REVIEW

File: libs/logging-bundle/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## Installation
+```shell
+composer require keboola/logging-bundle
+```
+
+## Configuration
+No configuration is required.
+
+Based on if the DataDog extension is available, the bundle will automatically register the log processor.
```
Comment (pivnicek) [inline libs/logging-bundle/README.md:13]: If I just looked at this readme I wouldn't know how to `## Usage` this bundle (of course I would suppose that I'd just register it in my services.yml, but maybe should we say so?

File: libs/logging-bundle/phpunit.xml.dist [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- https://phpunit.de/manual/current/en/appendixes.configuration.html -->
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
+         backupGlobals="false"
+         colors="true"
+         bootstrap="tests/bootstrap.php"
+         printerClass="Sempro\PHPUnitPrettyPrinter\PrettyPrinterForPhpUnit9"
```
Comment (pivnicek) [inline libs/logging-bundle/phpunit.xml.dist:8]: oooh, a new printer, excited to see the output :)

File: libs/logging-bundle/README.md (hunk 1 above)
Comment (joker806) [reply libs/logging-bundle/README.md:13]: No, you do not need to do anything at all, that's the beauty of the bundle :slightly_smiling_face: (in the current setup, maybe there will be some configuration in the future)

**Review by pivnicek — APPROVED** [review]
I get a little lost in the symfony stuff (like that kernel extension business, but i guess it's ok 😄

File: libs/logging-bundle/README.md (hunk 1 above)
Comment (pivnicek) [reply libs/logging-bundle/README.md:13]: I'm scared of magic

Comment (joker806) [reply libs/logging-bundle/README.md:13]: Not a magic, just something we were not used to yet :wink:

Comment (pivnicek) [reply libs/logging-bundle/README.md:13]: aha, so just composer install and voila, cool

File: libs/logging-bundle/tests/datadogStubs.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+// phpcs:disable
+if (function_exists('DDTrace\\current_context')) {
+    return;
+}
+// phpcs:enable
+
+function current_context(): array
+{
+    return [
```
Comment (odinuv) [inline libs/logging-bundle/tests/datadogStubs.php:15]: To nevypadá úplně jako stub

File: libs/logging-bundle/tests/KeboolaLoggingBundleTestingKernel.php [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+namespace Keboola\LoggingBundle\Tests;
+
+use Keboola\LoggingBundle\KeboolaLoggingBundle;
+use Symfony\Bundle\FrameworkBundle\FrameworkBundle;
+use Symfony\Bundle\MonologBundle\MonologBundle;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\HttpKernel\Kernel;
+
+class KeboolaLoggingBundleTestingKernel extends Kernel
```
Comment (odinuv) [inline libs/logging-bundle/tests/KeboolaLoggingBundleTestingKernel.php:13]: Doufám, že tohle je kompatibilní i se symfony 6, ať to pak neupdatujeme všechno nadvakrát.

File: libs/logging-bundle/tests/datadogStubs.php (hunk 3 above)
Comment (joker806) [reply libs/logging-bundle/tests/datadogStubs.php:15]: polyfill? nevim.. proste je to neco, co lokalne v testech fejkuje tu DD funkci

File: libs/logging-bundle/tests/KeboolaLoggingBundleTestingKernel.php (hunk 4 above)
Comment (joker806) [reply libs/logging-bundle/tests/KeboolaLoggingBundleTestingKernel.php:13]: cert vi, to se pozna :smile: ale uz vymyslim, jak to (jednoduse) testovat na vice verzich Symfony

**Pull request merged**